vcsclient/vcsclient.go

```diff
@@ -57,7 +57,7 @@ type VcsClient interface {
 	// payloadUrl    - URL to send the payload when a webhook event occurs
 	// token         - A token used to validate identity of the incoming webhook
 	// webhookId     - The webhook ID returned from a previous CreateWebhook command
-	// webhookEvents - PrCreated, PrEdited, or Push
+	// webhookEvents - PrOpened, PrEdited, or Push
```

Comment:
To avoid syncing this comment with the new events every time we add new ones:
```suggestion
	// webhookEvents - The event type
```